components/raftstore/src/store/worker/check_leader.rs

```diff
@@ -83,7 +83,7 @@ impl Runner {
                 meta.region_ranges
                 // get overlapped regions
                 .range((Excluded(start_key), Unbounded))
-                .take_while(|(_, id)| end_key > enc_start_key(&meta.regions[id]))
+                .take_while(|(_, id)| end_key > enc_start_key(&meta.regions[*id]))
```

Comment:
Why the change?

Reply:
Missed these comments.....Will change this back in the next PR.